### DISCUSSION TITLE
equivalent of `npm ci`?

### DISCUSSION THREAD

**Discussion by softloft38p-michael** [pnpm/pnpm#5948] [Q&A] [answered]
Is there a pnpm equivalent of `npm ci` (https://docs.npmjs.com/cli/v6/commands/npm-ci)?

**Comment by TheColorRed** [comment, marked as answer]
The equivalent is [`pnpm install --frozen-lockfile`](https://pnpm.io/cli/install#--frozen-lockfile)

**Reply by softloft38p-michael** [reply]
Thanks!
